Pipfile

```diff
@@ -47,6 +47,7 @@ pylint-django = "==2.0.11"
 pydocstyle = "==4.0.0"
 flake8-docstrings = "*"
 pyyaml = "*"
+tblib = "*"
```

Comment:
Thanks! Was about to do this when I saw my parallel test running made the Travis output unusable.